lib/src/summary.dart

```diff
@@ -126,6 +127,15 @@ class Summary {
 
     return map;
   }
+
+  PlatformSummary getPlatformSummary() {
+    Map<String, Platform> libraries = {};
+    dartFiles.values.forEach((dfs) {
+      libraries[dfs.uri] = dfs.platform;
```

Comment:
good point. I've updated to filter on `platform != null`, which atm. is the same as checking for only non-src files. However it is only a temp measure, instead, we shall introduce a flag on DartFileSummary, which should also track non-trivial `src` paths like `lib/something/src/internal.dart`.